### ISSUE TITLE
Errors when using WixUI_Mondo in wix 4.0.0

### ISSUE THREAD

**Issue by jobmarley** [wixtoolset/issues#7436]
If this issue is a bug:

* Which version of WiX are you building with?

4.0.0

* Which version of Visual Studio are you building with (if any)?

VS 2022 17.5.3

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

HeatWave 1.0.1

* Describe the problem and the steps to reproduce it.

> error WIX0204: ICE17: PushButton: 'Back' of Dialog: 'InstallDirDlg' does not have an event defined in the ControlEvent table. It is a 'Do Nothing' button.
> error WIX0204: ICE17: PushButton: 'ChangeFolder' of Dialog: 'InstallDirDlg' does not have an event defined in the ControlEvent table. It is a 'Do Nothing' button.

when using WixUI_Mondo with 
`<ui:WixUI Id="WixUI_Mondo" />`

* Describe the behavior you expected and how it differed from the actual behavior.

No errors

**Comment by jobmarley** [comment]
You can assign it to me if you want

**Comment by barnson** [comment]
Works for me. If you're using a customized dialog set, see https://wixtoolset.org/docs/fourthree/faqs/#converting-custom-wixui-dialog-sets.

**Comment by barnson** [comment]
Tests are at https://github.com/wixtoolset/wix/tree/develop/src/ext/UI/test/WixToolsetTest.UI.

**Comment by jobmarley** [comment]
Hey thanks for the reply.

I don't understand how it can work. WixUI_Mondo/Package.wxs match the code I have on my side, and it doesnt work either (just tried WixUI_Mondo/Package.wxs, didnt run compilation+tests).  

Looks like the problem is related to this https://github.com/wixtoolset/wix/commit/c4ffb9967e4e4f7d67f5925208e3e0a885e903c5#diff-3c612e3656c86b9a7677bba2b03475852bf44246e1cf1d13712b192461004390.
Mondo is the only file missing Next/Back controls for InstallDirDlg. That corresponds to the errors I get.

That seems weird that nobody noticed that since december, but the errors align with the code.  

I don't know the project so feel free to hit me up if I missed anything, so I don't end up commiting a useless fix.

**Comment by barnson** [comment]
Yeah, so WixUI_Mondo doesn't use InstallDirDlg so there shouldn't be a reference to it to add the DoAction control event. I fixed that and ran a validation pass for the fun of it.

**Comment by barnson** [comment]
Reopening for triage.
